README.md

```diff
@@ -128,38 +128,38 @@ interface which abstracts over both. It is parameterized with a
 trait, which allows users to specify what underlying implementation they want to use.
 
 Once we've
-[initialized the JIT data structures](https://github.com/sunfishcode/simplejit-demo/blob/master/src/jit.rs#L29),
+[initialized the JIT data structures](./src/jit.rs#L29),
 we then use our `JIT` to
-[compile](https://github.com/sunfishcode/simplejit-demo/blob/master/src/jit.rs#L46)
+[compile](./jit.rs#L46)
```

Comment:
nit: `src/jit.rs`

Reply:
Sorry about my late reply. I've added a commit with this now. Thanks